### PR TITLE
FAD-6423 Collect payments after updating payment info

### PR DESCRIPTION
**Pull request by rajumsys**
How to test:
- Go to /billing; open your console
- Click `Update Payment Information`
- Put the new credit card info
- Submit the form. You should see a request to `/collect` endpoint

Notes:
- Contrary to what ticket said, seems api doesn't respond with 400 when nothing to be collected. Seems API now responds a 200 with a description. [See](https://github.com/SparkPost/accusers-api/blob/master/api/resources/account-controller.js#L878-L880)
- [Test credit cards](https://developer.merchante-solutions.com/#/payment-gateway-testing#validation)

### CODE REVIEW

File: src/actions/billing.js [hunk 1]
```diff
@@ -147,6 +147,21 @@ export function billingCreate(values) {
   };
 }
 
+/**
+ * attempts to collect payments (like when payment method is updated) to make sure pending payments are charged
+ * should suppress 400 from api as they are good here (means nothing to collect)
```
Comment (jasonrhodes) [inline src/actions/billing.js]: remove this part of the comment now, yay

**Comment by jasonrhodes** [comment]
One small change to remove unnecessary comment, otherwise LGTM!

File: src/actions/billing.js (hunk 1 above)
Comment (rajumsys) [reply src/actions/billing.js]: what? i removed that! how come it's still here!

Comment (rajumsys) [reply src/actions/billing.js]: ok, i haven't (force) pushed earlier

**Pull request closed without merging**